cfgov/core/tests/test_middleware.py

```diff
@@ -122,3 +122,15 @@ def test_non_default_encoding(self):
         encoding = 'gb2312'
         parsed = parse_links(s.encode(encoding), encoding=encoding)
         self.assertEqual(parsed, s)
+
+    def test_external_link_with_attribute(self):
+        s = '<a href="https://somewhere/foo" data-thing="something">Link</a>'
+        output = parse_links(s)
+        self.assertIn('external-site', output)
+        self.assertIn('cf-icon-svg', output)
+
+    def test_external_link_with_img(self):
+        s = '<a href="https://somewhere/foo"><img src="some.png"></a>'
+        output = parse_links(s)
+        self.assertIn('external-site', output)
+        self.assertIn('cf-icon-svg', output)
```

Comment:
Is this actually the desired behavior, to add a link icon to image links?

Reply:
You're right — it's probably not.

Reply:
Fixed in c654196.